scripts/easyphoto_tryon_infer.py

```diff
@@ -0,0 +1,720 @@
+                f"--dataloader_num_workers={dataloader_num_workers}",
+                f"--max_train_steps={max_train_steps}",
+                f"--checkpointing_steps={val_and_checkpointing_steps}",
+                f"--learning_rate={learning_rate}",
+                "--lr_scheduler=constant",
+                "--lr_warmup_steps=0",
+                "--train_text_encoder",
+                "--seed=42",
+                f"--rank={rank}",
+                f"--network_alpha={network_alpha}",
+                f"--output_dir={os.path.relpath(weights_save_path, pwd)}",
+                f"--logging_dir={os.path.relpath(weights_save_path, pwd)}",
+                "--enable_xformers_memory_efficient_attention",
+                "--mixed_precision=fp16",
+                f"--cache_log_file={cache_log_file_path}",
+            ]
+            try:
+                subprocess.run(command, check=True)
+            except subprocess.CalledProcessError as e:
+                ep_logger.error(f"Error executing the command: {e}")
+
+        else:
+            command = [
+                f"{python_executable_path}",
+                "-m",
+                "accelerate.commands.launch",
+                "--mixed_precision=fp16",
+                "--main_process_port=3456",
+                f"{train_kohya_path}",
+                f"--pretrained_model_name_or_path={sd15_save_path}",
+                f"--pretrained_model_ckpt={webui_load_path}",
+                f"--train_data_dir={user_path}",
+                "--caption_column=text",
+                f"--resolution={resolution}",
+                f"--train_batch_size={train_batch_size}",
+                f"--gradient_accumulation_steps={gradient_accumulation_steps}",
+                f"--dataloader_num_workers={dataloader_num_workers}",
+                f"--max_train_steps={max_train_steps}",
+                f"--checkpointing_steps={val_and_checkpointing_steps}",
+                f"--learning_rate={learning_rate}",
+                "--lr_scheduler=constant",
+                "--lr_warmup_steps=0",
+                "--train_text_encoder",
+                "--seed=42",
+                f"--rank={rank}",
+                f"--network_alpha={network_alpha}",
+                f"--output_dir={weights_save_path}",
+                f"--logging_dir={weights_save_path}",
+                "--enable_xformers_memory_efficient_attention",
+                "--mixed_precision=fp16",
+                f"--cache_log_file={cache_log_file_path}",
+            ]
+            try:
+                subprocess.run(command, check=True)
+            except subprocess.CalledProcessError as e:
+                ep_logger.error(f"Error executing the command: {e}")
+
+        best_weight_path = os.path.join(weights_save_path, f"pytorch_lora_weights.safetensors")
+        if not os.path.exists(best_weight_path):
+            return ("Failed to obtain Lora after training, please check the training process.", [], template_mask, reference_mask)
+
```

Comment:
I got an error occurred here because the target folder does not exist. Following suggested code should be added.

```suggestion
        os.makedirs(cloth_gallery_dir, exist_ok=True)
```

Reply:
OK，thanks